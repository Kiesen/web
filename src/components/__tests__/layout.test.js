import React from 'react'
import renderer from 'react-test-renderer'

import Layout from 'src/components/layout'

describe('The layout component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Layout />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
