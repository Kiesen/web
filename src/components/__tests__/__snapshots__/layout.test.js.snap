// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The layout component renders correctly 1`] = `
<div
  className="sc-bdVaJa UkKX"
/>
`;
